esteid/middleware.py

```diff
@@ -69,9 +119,38 @@ def get_ocsp_responder_certificate_path(cls):
         return config.ocsp_responder_certificate_path()
 
     @classmethod
-    def verify_ocsp(cls, request):
-        issuer = request.META.get('HTTP_X_ISSUER', None)
-        certificate = request.META.get('HTTP_X_CLIENTCERT', None)
+    def verify_ocsp(cls, certificate, issuer):
+        """ Runs OCSP verification and returns error code - 0 means success
+        """
+
         return OCSPVerifier(certificate, issuer,
```

Comment:
What do you think about padding the ocsp error numbers as well?

Reply:
Not sure about padding, how about creating an enum of possible error codes and returning values of that instead? Would also make checks more readable.

Expanding on this idea, other error codes ala `ERROR_NO_DN` could be similarily converted into enum.
Your thoughts?

Reply:
It probably wouldn't then. Ok, how would you do the OCSP padding? Add... 4000 to them?

Reply:
Yup